packages/ckeditor5-self-request/src/ui/linkactionsview.ts

```diff
@@ -245,13 +267,6 @@ export default class LinkActionsView extends View {
 
 		return button;
 	}
-
-	destroy() {
```

Comment:
Why are you removing the unsubscribe ?

Reply:
I didn't remove it, it got moved to ``public override destroy(): void``, please check [here](https://github.com/gymglish/ckeditor5-build-inline/blob/45146b4ef33f3a0152d9953b7de47826dc661a2a/packages/ckeditor5-self-request/src/ui/linkactionsview.ts#L169-L171)